src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,27 @@
 // Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo:React.FC<Props> = ({ todo }) => {
+  const { completed, title, user } = todo;
+
+  return (
+    <>
+      <article className={`TodoInfo ${completed ? 'TodoInfo--completed' : ''}`}>
+        <h2 className="TodoInfo__title">
+          { title }
+        </h2>
+
+        {user && (
+          <UserInfo user={user} />
+        )}
+      </article>
+    </>
```

Comment:
Why did you use react fragment here?

```suggestion
      <article className={`TodoInfo ${completed ? 'TodoInfo--completed' : ''}`}>
        <h2 className="TodoInfo__title">
          { title }
        </h2>

        {user && (
          <UserInfo user={user} />
        )}
      </article>
```